tests/_files/ExceptionInTearDownAfterClassTest.php

```diff
@@ -0,0 +1,62 @@
+<?php
+/*
+ * This file is part of PHPUnit.
+ *
+ * (c) Sebastian Bergmann <sebastian@phpunit.de>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+use PHPUnit\Framework\TestCase;
+
+class ExceptionInTearDownAfterClassTest extends TestCase
+{
+    public $setUp                = false;
```

Comment:
Is there a specific reason why these attributes are `public`?

Reply:
No anymore no, thanks for catching this. The test started off as a version of `tests/_files/ExceptionInTearDownTest.php`. I have cleaned up test to the minimum required.